cla_backend/apps/knowledgebase/tests/test_csv_2_fixture.py

```diff
@@ -28,1416 +47,1446 @@ def slice_records(self, output_records, expected_records, start_index=None, end_
+                        u"created": u"{date}".format(date=self.datetime_now),
+                        u"modified": u"{date}".format(date=self.datetime_now),
+                        u"article": 2,
+                        u"article_category": 16,
+                        u"preferred_signpost": False,
+                    },
+                },
+                {
+                    u"pk": 35,
+                    u"model": u"knowledgebase.articlecategorymatrix",
+                    u"fields": {
+                        u"created": u"{date}".format(date=self.datetime_now),
+                        u"modified": u"{date}".format(date=self.datetime_now),
+                        u"article": 2,
+                        u"article_category": 17,
+                        u"preferred_signpost": True,
+                    },
+                },
+                {
+                    u"pk": 36,
+                    u"model": u"knowledgebase.articlecategorymatrix",
+                    u"fields": {
+                        u"created": u"{date}".format(date=self.datetime_now),
+                        u"modified": u"{date}".format(date=self.datetime_now),
+                        u"article": 2,
+                        u"article_category": 18,
+                        u"preferred_signpost": True,
+                    },
+                },
+            ]
+            outputList = json.loads(outputJSON)
 
-        self.compare_length(outputList, expectedList)
+            self.compare_length(outputList, expectedList)
 
-        output_article_categories, expected_article_categories = self.slice_records(
-            outputList, expectedList, end_index=18
-        )
-        self.compare_list_of_records(output_article_categories, expected_article_categories)
+            # Refactor
+            output_article_categories, expected_article_categories = self.slice_records(
+                outputList, expectedList, end_index=18
+            )
+            self.compare_list_of_records(output_article_categories, expected_article_categories)
 
-        output_article_1, expected_article_1 = self.slice_records(outputList, expectedList, 18, 19)
-        self.compare_list_of_records(output_article_1, expected_article_1)
+            output_article_1, expected_article_1 = self.slice_records(outputList, expectedList, 18, 19)
+            self.compare_list_of_records(output_article_1, expected_article_1)
 
-        output_acm_1, expected_acm_1 = self.slice_records(outputList, expectedList, 19, 37)
-        pk_range_acm_1 = self.calculate_pk_range(output_acm_1)
-        output_sorted_acm_1, expected_sorted_acm_1 = self.sort_article_category_matrices(output_acm_1, expected_acm_1)
-        self.compare_article_category_matrices(output_sorted_acm_1, expected_sorted_acm_1, pk_range_acm_1)
+            output_acm_1, expected_acm_1 = self.slice_records(outputList, expectedList, 19, 37)
+            pk_range_acm_1 = self.calculate_pk_range(expected_acm_1)
```

Comment:
calculate_pk_range only takes in 1 article category matrix

Reply:
No, this is still a list of `CategoryMatrix` objects. Min and max pks wouldn't make sense if it was just one object, would it?